packages/cli/src/cli.ts

```diff
@@ -11,11 +12,19 @@ import { setupCliContainer } from './setup-cli-container';
 
 (async () => {
     const scanArguments = (yargs
-        .usage('Usage: $0 --url <url> --output <directoryPath>')
+        .usage('Usage: $0 --url <url> --filePath <filePath> --output <directoryPath>')
         .options({
-            url: { type: 'string', describe: 'url to scan for accessibility', demandOption: true },
+            url: { type: 'string', describe: 'url to scan for accessibility' },
+            filePath: { type: 'string', describe: 'file path that contians multiple urls to scan' },
             output: { type: 'string', describe: 'output directory' },
         })
+        .check((args) => {
+            if ((isEmpty(args.url) && isEmpty(args.filePath)) || (!isEmpty(args.url) && !isEmpty(args.filePath))) {
+                throw new Error('You should input either url or filePath (only one)!');
```

Comment:
Will user see error stack when error is thrown? May be user message can be like `You should provide either url or filePath parameter only`

Reply:
he'll see the error message, because we have a catch block that prints it then exits